app/client/components/dashboard/card/GroupByChart.jsx

```diff
@@ -20,7 +20,10 @@ class GroupByChart extends React.Component {
 
     const aggregate = this.props.data.aggregate;
     const aggregateKeys = aggregate ? Object.keys(aggregate) : [];
-    const aggregateData = aggregate ? Object.values(aggregate) : [];
+    var aggregateData = aggregate ? Object.values(aggregate) : [];
+    if (this.props.data.type.type == "currency") {
```

Comment:
1) `this.props.data.type.type` seems a bit messy. Can we be more explicit?
2) don't love the string comparison, but not sure if there is a neater way to do it

Reply:
1. Changed to `value_type`  
2. Made it a const, so it's a little bit cleaner now